src/pages/PullRequestPage/PullBundleAnalysis/ErrorBanner/ErrorBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import { ComparisonReturnType } from 'shared/utils/comparison'
+import A from 'ui/A'
+import Banner from 'ui/Banner'
+
+import { TBundleAnalysisComparisonResult } from '../../hooks'
+
+interface Props {
+  errorType?: TBundleAnalysisComparisonResult
+}
+
+const BannerContent: React.FC<Props> = ({ errorType }) => {
+  if (errorType === ComparisonReturnType.MISSING_BASE_COMMIT) {
+    return (
+      <>
+        {' '}
+        <h1 className="font-semibold">Missing Base Commit</h1>
+        <div className="flex gap-1">
+          <span>
+            Unable to compare commit because no base commit was found.
```

Comment:
nit: Do we want this to be `commit` or `commits`?

Reply:
I think `commit` makes sense here because there can only be one base commit ... right?